_posts/2020-03-30-community-update.md

```diff
@@ -0,0 +1,130 @@
+
+Throw into the mix improvements around type systems, the WebUI, metrics reporting and supported formats, this release is bound to keep the community busy. For a complete overview of the ongoing development, check [this discussion](http://apache-flink-mailing-list-archive.1008284.n3.nabble.com/DISCUSS-Features-of-Apache-Flink-1-11-td38724.html#a38793) and follow the weekly updates on the Flink [@community mailing list](https://flink.apache.org/community.html#mailing-lists).
+	
+## New Committers and PMC Members
+
+The Apache Flink community has welcomed **1 PMC (Project Management Committee) Member** and **5 new Committers** since the last update (September 2019):
+
+### New PMC Members
+	Jark Wu
+
+### New Committers
+	Zili Chen, Jingsong Lee, Yu Li, Dian Fu, Zhu Zhu
+
+Congratulations to all and thank you for your hardworking commitment to Flink!
+
+# The Bigger Picture
+
+## A Look into the Flink Repository
+
+In the [last update](https://flink.apache.org/news/2019/09/10/community-update.html), we showed off some numbers around Flink releases and mailing list activity. This time, we’re looking into the activity in the Flink repository and how it’s evolving.
+
+<center>
+<img src="{{ site.baseurl }}/img/blog/2020-03-30-flink-community-update/2020-03-30-flink-community-update_1.png" width="725px" alt="GitHub 1"/>
+</center>
+
+There is a clear upward trend in the number of contributions to the repository, based on the number of commits. This reflects the **fast pace of development** the project is experiencing and also the **successful integration of the China-based Flink contributors** started early last year. To complement these observations, the repository registered a **1.5x increase in the number of individual contributors in 2019**, compared to the previous year. 
+
+But did this increase in capacity produce any other measurable benefits?
+
+<center>
+<img src="{{ site.baseurl }}/img/blog/2020-03-30-flink-community-update/2020-03-30-flink-community-update_2.png" width="725px" alt="GitHub 2"/>
+</center>
+
+If we look at the average time of Pull Request (PR) “resolution”, it seems like it did: **the average time it takes to close a PR these days has been steadily decreasing** since last year, sitting between 5-6 days for the past few months. 
```

Comment:
The graphs seem a bit inconsistent?
Why are 2018 and 2019 shown as bars, and 2020 is shown as a line? (in the contributors chart)
For the PR chart, the roles assignment of lines and bars has changed again.
Why can't use use bars for all years? Or at least use the line always for the current year?

Reply:
The idea was to make the chart as easy to interpret as possible, which was not the case with the formatting you mention (I tried a couple of different customisations and charts). For the PR data, for example, it's more valuable to see the "worst performance" isolated to contrast with where we are today than to have today's values lost in the lower end of the other bars.

<img width="802" alt="Screen Shot 2020-03-31 at 10 28 50" src="https://user-images.githubusercontent.com/23521087/78004533-854dfe80-733a-11ea-8ab0-a34321c8c4b0.png">

Let me know if this makes sense to you.

Reply:
Yes, this is better, thanks :)